Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN npm install -g yarn
 COPY package.json ./
 COPY yarn.lock ./
 RUN yarn install
+RUN yarn add jest
```

Comment:
Oops, we can probably revert this one...

Reply:
Reverted and built assets